src/schemas/components/SchemaFormProperties.vue

```diff
@@ -45,23 +36,19 @@
   import SchemaFormProperty from '@/schemas/components/SchemaFormProperty.vue'
   import SchemaFormPropertyAllOf from '@/schemas/components/SchemaFormPropertyAllOf.vue'
   import SchemaFormPropertyAnyOf from '@/schemas/components/SchemaFormPropertyAnyOf.vue'
-  import { useSchema } from '@/schemas/compositions/useSchema'
   import { SchemaProperty, SchemaProperties, isPropertyWith } from '@/schemas/types/schema'
   import { SchemaValues } from '@/schemas/types/schemaValues'
-  import { getSchemaDefinition } from '@/schemas/utilities/definitions'
 
   const props = defineProps<{
     parent: SchemaProperty,
     properties: SchemaProperties,
-    values: SchemaValues,
+    values: SchemaValues | undefined,
```

Comment:
Values can always be undefined because a value might not exist yet on the form's v-model